x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/feature_flag/infrastructure.ts

```diff
@@ -59,12 +59,12 @@ describe('Infrastracture feature flag', () => {
 
       cy.get(infraToggle)
         .should('have.attr', 'aria-checked')
-        .and('equal', 'false');
+        .and('equal', 'true');
     });
 
-    it('hides infrastructure tab in service overview page', () => {
+    it('shows infrastructure tab in service overview page', () => {
       cy.visit(serviceOverviewPath);
-      cy.contains('a[role="tab"]', 'Infrastructure').should('not.exist');
+      cy.contains('a[role="tab"]', 'Infrastructure').click();
```

Comment:
Why is this click() there? I get that you swapped things around, wondering why it was there in the first place. Is it needed? Is it a test masked as an interaction?

Reply:
Good point, I think it's a test. If that's the case we could improve it during the test plan @MiriamAparicio what do you think?

Reply:
Added it in